### PR TITLE
Portfolio finish mobile version form

### PR DESCRIPTION
**Pull request by Juanpa8830**
# Hi reviewer.
## These are the proposed changes for the project:

- Add the form section.
- Add the footer.
- Add links to social media icons.
- Collaborate with coding partner. @Khawanin

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -166,11 +169,56 @@ <h4>Skills</h4>
...
             </ul>
         </section>
-        
+
+        <section id="greetings2">
+            <h1 class="GE">Get started</h1>
+            <p class="firstp">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Atque doloribus maiores,
+                rem excepturi.</p>
+
+                 <form class="form" action="#" target="" method="get" name="formDatosPersonales">
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] please ensure that your Form uses the action [formspree service](https://formspree.io/html).
eg action value should be similar to https://formspree.io/f/{form_id}
`method` value equals `POST`
check more [here](https://formspree.io/html/)

File: index.html [hunk 2]
```diff
@@ -166,11 +169,56 @@ <h4>Skills</h4>
...
+                    <ul>
+                        <li><label for="text">Your name</label>
+                        <input type="text" name="nombre" id="nombre" required/>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] please ensure your name input has `maxlength="30"` attribute. this will limit long names

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Juanpa8830 
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @Juanpa8830 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**